tests/reference_datatype_tests.py

```diff
@@ -189,7 +189,7 @@ def test_get_display_value(self):
                 "data": {
                     "72048cb3-adbc-11e6-9ccf-14109fd34195": [
                         {
-                            "uri": "https://rdm.dev.fargeo.com/plugins/controlled-list-manager/item/9baf3cd5-33d4-4fbc-b1d1-a2d218732f1e",
+                            "uri": "https://lingo.dev.fargeo.com/plugins/controlled-list-manager/item/9baf3cd5-33d4-4fbc-b1d1-a2d218732f1e",
```

Comment:
relying on an external url is probably not the most stable test logic.

Reply:
it could be a totally fake url, I just didn't want to suggest there was anything at this location